### PR TITLE
Drop support for PHP 7.3

### PR DESCRIPTION
**Pull request by core23**
## Subject

Drop support for PHP 7.3, because active support will end in a few days. This allows us to add type hints everywhere.

I am targeting this branch, because this is a major change.

## Changelog

```markdown
### Changed
- Added type hints to every property and method.

### Removed
- Dropped support for PHP 7.3

```

### CODE REVIEW

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -25,15 +25,14 @@ jobs:
         strategy:
             matrix:
                 php-version:
-                    - '7.3'
```
Comment (VincentLanglet) [inline .github/workflows/test.yaml:28]: This should be on dev-kit

File: src/Event/BreadcrumbListener.php [hunk 2]
```diff
@@ -24,10 +24,7 @@
  */
 final class BreadcrumbListener
 {
-    /**
-     * @var array
-     */
-    private $blockServices = [];
+    private array $blockServices = [];
```
Comment (VincentLanglet) [inline src/Event/BreadcrumbListener.php:27]: We can still add phpdoc `array<...>`

**Review by VincentLanglet — APPROVED** [review]

File: src/Event/BreadcrumbListener.php (hunk 2 above)
Comment (core23) [reply src/Event/BreadcrumbListener.php:27]: Will check this when increasing phpstan / psalm level

**Comment by jordisala1991** [comment]
Do you think it is a good idea for next majors to have different php required versions?

**Comment by VincentLanglet** [comment]
> Do you think it is a good idea for next majors to have different php required versions?

I don't see any issue with this. You can still bump Php and then the major version.
Anyway, the major will be release potentially after the php version drop.

**Pull request merged**